### PR TITLE
Adding DeclStmt.Decls

### PR DESCRIPTION
**Pull request by udaken**
Added missing API.

### CODE REVIEW

File: sources/ClangSharp/Cursors/Stmts/DeclStmt.cs [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 {
     public sealed class DeclStmt : Stmt
     {
+        private readonly Lazy<IReadOnlyList<Decl>> _children;
         internal DeclStmt(CXCursor handle) : base(handle, CXCursorKind.CXCursor_DeclStmt)
         {
+            _children = new Lazy<IReadOnlyList<Decl>>(() => CursorChildren.OfType<Decl>().ToList());
         }
+
+        public IReadOnlyList<Decl> Decls => _children.Value;
```
Comment (tannergooding) [inline sources/ClangSharp/Cursors/Stmts/DeclStmt.cs]: Would be good to also expose `IsSingleDecl` (`isSingleDecl`) and `SingleDecl` (`getSingleDecl`)

File: sources/ClangSharp/Cursors/Stmts/DeclStmt.cs [hunk 2]
```diff
@@ -1,11 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
 using ClangSharp.Interop;
 
 namespace ClangSharp
 {
     public sealed class DeclStmt : Stmt
     {
+        private readonly Lazy<IReadOnlyList<Decl>> _children;
```
Comment (tannergooding) [inline sources/ClangSharp/Cursors/Stmts/DeclStmt.cs]: nit: Missing newline between field and constructor

Comment (tannergooding) [inline sources/ClangSharp/Cursors/Stmts/DeclStmt.cs]: nit: The field should be named `_decls` to be consistent with other usages.

**Comment by udaken** [comment]
@tannergooding 
Sorry for delay.
thank you for feedback.
i fixed it.

**Comment by tannergooding** [comment]
Thanks!

**Pull request merged**